h2o-py/h2o/automl/autoh2o.py

```diff
@@ -275,11 +280,30 @@ def leaderboard(self):
         """
         return H2OFrame([]) if self._leaderboard is None else self._leaderboard
 
+    @property
+    def event_log(self):
+        """
+        retrieve the backend event log from an H2OAutoML object
+
+        :return: an H2OFrame with detailed events occurred during the AutoML training.
+        """
+        return H2OFrame([]) if self._event_log is None else self._event_log
+
+    @property
+    def training_info(self):
```

Comment:
@ledell feel free to come up with a better name, as this may come handy to expose more info in the future for post-analysis

Reply:
@seb-h2o What about something more generic, e.g. `event_log()` or `log_info()`?  Using the term "training" to me, its not immediately clear whether that refers to the training set or model training.

Reply:
@ledell  I use `event_log` property right above for the full EventLog frame: 6 cols, n rows.
This `training_info` is just sugar on top: it extracts log entries with key-values as a dictionary for ease of use. Those key-values are mainly useful for post-analysis as we discussed: https://0xdata.atlassian.net/browse/PUBDEV-5316

Reply:
`log_info` maybe. I'm not happy with `training_info` either: I was mainly looking for a word that  says it contains meta information about the training that can be useful for post-analysis.